packages/core/src/tracing/cls.js

```diff
@@ -69,6 +69,8 @@ function init(config, _processIdentityProvider) {
  * @property {{s?: number, d?: number}} [b]
  * @property {*} [f]
  * @property {*} [gqd]
+ * @property {*} [error]
+ * @property {*} [async]
```

Comment:
I can see where this is coming from (I think) but we definitely should not add these two attributes to this type! 

They have been deprecated and removed from the production code a long time ago and they should never be present on any spans we produce, so we should not allow for them in this type.

When removing them from the code base I also added expect(span.error).to.not.exist and expect(span.async).to.not.exist to a lot of tests just to make sure I hadn't forgotten to remove them everywehere. Then, when copying those verification methods around, that pattern spread to a lot of test cases. My bad, I screwed that up, I should have removed those checks everywhere like two years ago. :-( 

You probably added those two here to get the new verifyXxx methods to typecheck. Instead of adding these two attributes to the type, we should instead remove the checks for the non-existence, at least in the new type checked verifyXxx methods. We know that we do not create these attributes anywhere, keeping these two checks around has no value whatsoever.